packages/mdc-typography/README.md

```diff
@@ -26,145 +38,71 @@ developers as CSS classes.
-
-The `mdc-typography` class defines basic properties for text, such as the typeface and antialiasing settings.
-
-### Formatting your text
-
-#### Style
-
-Simply add the corresponding style class to format your text:
-
-```html
-<body class="mdc-typography">
-  <h1 class="mdc-typography--display4">Big header</h1>
-
-  <p class="mdc-typography--body1">
-    A paragraph with <span class="mdc-typography--body2">emphasis</span>.
-  </p>
-</body>
+npm install --save @material/typography
 ```
 
-The full list of styles:
-
-- `mdc-typography--display4`
-- `mdc-typography--display3`
-- `mdc-typography--display2`
-- `mdc-typography--display1`
-- `mdc-typography--headline`
-- `mdc-typography--title`
-- `mdc-typography--subheading2`
-- `mdc-typography--subheading1`
-- `mdc-typography--body2`
-- `mdc-typography--body1`
-- `mdc-typography--caption`
-
-#### Margins and positioning
+## Usage
 
-In order to minimize unexpected behavior, the style classes only specify font properties, such as size, weight and line
-height.
+### Load Roboto
 
-This means that while text will be correctly styled, it may not be correctly positioned. If you include the
-`mdc-typography--adjust-margin` class, though, positioning will be adjusted according to the style:
+We recommend you load Roboto from Google Fonts
 
 ```html
-<body class="mdc-typography">
-  <h1 class="mdc-typography--display4 mdc-typography--adjust-margin">
-    Big header
-  </h1>
-
-  <p class="mdc-typography--body1 mdc-typography--adjust-margin">
-    A paragraph with
-    <span class="mdc-typography--body2 mdc-typography--adjust-margin">
-      emphasis
-    </span>.
-  </p>
-</body>
-```
-
-> Note: as the name implies, `mdc-typography--adjust-margin` will change the margin properties of the element it's
-applied to, in order to align text correctly. Because of this, it should only be used in a text context; using this
-property on UI elements such as buttons may cause them to be positioned incorrectly.
-
-
-## Sass mixin usage
-
-### mdc-typography-base
-
-```scss
-@include mdc-typography-base;
-```
-
-`mdc-typography-base` defines the basic properties for Material Design typography, namely the font and aliasing
-settings, without defining any particular font size or style.
-
-
-### mdc-typography
-
-```scss
-@include mdc-typography(display4);
-```
-
-Applies one of the typography styles. Note that this includes the font family and aliasing definitions; you don't need
-to include `mdc-typography-base` as well.
-
-The full list of styles:
-- `display4`
-- `display3`
-- `display2`
-- `display1`
-- `headline`
-- `title`
-- `subheading2`
-- `subheading1`
-- `body2`
-- `body1`
-- `caption`
-
-
-### mdc-typography-adjust-margin
-
-```scss
-@include mdc-typography(display4);
-@include mdc-typography-adjust-margin(display4);
+<head>
+  <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500" rel="stylesheet">
+</head>
 ```
```

Comment:
Also consider moving this example higher up in the document. Possibly directly above `## Design & API Documentation`